client/app/components/AceEditorInput.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef } from 'react';
+import AceEditor from 'react-ace';
+
+import './AceEditorInput.less';
+
+function AceEditorInput(props, ref) {
+  return (
+    <div className="ace-editor-input">
+      <AceEditor
+        ref={ref}
+        mode="sql"
+        theme="textmate"
+        height="100px"
+        editorProps={{ $blockScrolling: Infinity }}
+        {...props}
```

Comment:
```suggestion
        showPrintMargin={false}
        {...props}
```

So this unnecessary line doesn't show.
<img width="478" alt="Screen Shot 2019-07-12 at 10 13 09" src="https://user-images.githubusercontent.com/486954/61109493-af4e5080-a48d-11e9-9c8c-d34327decb8a.png">